pkgs/applications/science/biology/poretools/default.nix

```diff
@@ -0,0 +1,22 @@
+{ stdenv, pythonPackages, fetchgit }:
+
+pythonPackages.buildPythonPackage rec {
+  pname = "poretools";
+  version = "0.6.0";
+  name = "${pname}-${version}";
+
+  src = fetchgit {
+    url = "git://github.com/arq5x/${pname}";
```

Comment:
Does this repository contain submodules? If yes, please use `https://` for fetchgit.
if no, then please switch to `fetchFromGitHub`.